locations/philmel.yml

```diff
@@ -0,0 +1,198 @@
+---
+location: philmel
+location_nice: Philipp-Melanchthon-Kirche
+latitude: 52.465881
+longitude: 13.434112
+contact_nickname: 'roedel'
+contacts:
+  - '@roedel:matrix.org'
+
+hosts:
+
+  - hostname: philmel-core
+    role: corerouter
+    model: "linksys_e8450-ubi"
+    wireless_profile: freifunk_default
+
+  - hostname: philmel-nf-east-5ghz
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    wireless_profile: freifunk_default
+    mac_override:
+      eth0: cc:2d:e0:9c:4f:00
+
+  - hostname: philmel-nf-s-5ghz
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    wireless_profile: freifunk_default
+    mac_override:
+      eth0: cc:2d:e0:9c:4f:00
+
+snmp_devices:
+
+  - hostname: philmel-rhnk
+    address: 10.230.2.4
+    snmp_profile: af60
+
+  - hostname: philmel-nord-5ghz
+    address: 10.230.2.6
+    snmp_profile: airos_8
+
+  - hostname: philmel-ost
+    address: 10.230.2.7
+    snmp_profile: airos_6
+
+  - hostname: philmel-vaterhaus
+    address: 10.230.2.8
+    snmp_profile: airos_8
+
+  - hostname: philmel-sued
+    address: 10.230.2.9
+    snmp_profile: airos_6
+
+  - hostname: philmel-sued-5ac
+    address: 10.230.2.10
+    snmp_profile: airos_8
+
+  - hostname: philmel-west
+    address: 10.230.2.11
+    snmp_profile: airos_6
+
+  - hostname: philmel-ak36
+    address: 10.230.2.12
+    snmp_profile: airos_8
+
+# TODO: Add all devices and
+# change naming
+airos_dfs_reset:
+  - name: "philmel-ak36"
+    target: "10.230.2.12"
+    username: "ubnt"
+    password: "file:/root/pwd.txt"
+    daytime_limit: "2-7"
+  - name: "philmel-nord-5ac"
+    target: "10.230.2.6"
+    username: "ubnt"
+    password: "file:/root/pwd.txt"
+    daytime_limit: "2-7"
+
+ipv6_prefix: "2001:bf7:820:1500::/56"
+
+# Complete subnet: 10.230.2.0/24
+# MGMT Network: 10.230.2.0/28
+# Mesh Network: 10.230.2.16/28 (-23)
+# DHCP Network: 10.230.2.32/28
+
+networks:
+  - vid: 2
```

Comment:
Based on the convention for VLANs this VID should be 40: https://github.com/freifunk-berlin/bbb-configs/blob/master/DEVELOPER.md#:~:text=The%20VLAN%20ID%20(vid)%20usually%20follow%20this%20numbering%20convention.